packages/sdk-vue/src/FusionAuthVuePlugin.ts

```diff
@@ -0,0 +1,53 @@
+import { App } from 'vue';
+import { FusionAuth, FusionAuthConfig } from './types.ts';
+import * as components from './components/index.ts';
+import { fusionAuthKey } from './injectionSymbols.ts';
+import { createFusionAuth } from './createFusionAuth/index.ts';
+
+type FusionAuthInstantiated = { instance: FusionAuth };
+
+/**
+ * Installation method for the FusionAuthVuePlugin.
+ *
+ * @category Plugin
+ * @param {App} app - The Vue app instance.
+ * @param {FusionAuthConfig | FusionAuthInstantiated} options - The configuration options for the plugin or an object containing a FusionAuth instance.
+ * @throws {Error} Will throw an error if the required options are missing.
+ */
+const FusionAuthVuePlugin = {
+  install(app: App, options: FusionAuthConfig | FusionAuthInstantiated) {
+    let fusionAuth: FusionAuth;
+
+    if ('instance' in options) {
+      fusionAuth = options.instance;
+    } else {
+      const config = options as FusionAuthConfig;
```

Comment:
With the type guard in place we can probably remove this line and just pass in 'options' instead of config since we have handling for which options we are referencing.

Reply:
TS yells if we remove that 😑